src/components/Modal/style.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+export const WrapperSt = styled.div`
```

Comment:
С пустой строкой тут лучше читается
```suggestion
import styled from 'styled-components';

export const WrapperSt = styled.div`
```

Вообще можно на это дело линтер настроить